hack/make.sh

```diff
@@ -333,6 +333,19 @@ copy_containerd() {
 	fi
 }
 
+install_binary() {
+	file="$1"
+	target="${DOCKER_MAKE_INSTALL_PREFIX:=/usr/local}/bin/"
+	if [ "$(go env GOOS)" == "linux" ]; then
+		echo "Installing $(basename $file) to ${target}"
+		cp $file $target
```

Comment:
(but otherwise, the rest of the PR still looks great, so consider this a "re-LGTM" :innocent:)